packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -62,6 +63,12 @@ export default class LocusInfo extends EventsScope {
     // this.info = locus.info || null;
   }
 
+  /**
```

Comment:
Descriptions are needed in these headers.

Reply:
this is a private class, not used externally